website/docs/api/indexing.mdx

```diff
@@ -81,7 +81,7 @@ Returns 0 (`E_SUCCESS`) on success.
 <TabItem value="c">
 
 ```c
-H3Error cellToPoint(H3Index cell, GeoPoint *g);
+H3Error cellToLatLng(H3Index cell, LatLng *g);
```

Comment:
missing update to Python

Reply:
I was holding off as we haven't made the Python changes yet, but I can put a placeholder here.